frontend/public/components/catalog-item-icon.tsx

```diff
@@ -196,9 +204,17 @@ export type ClusterServiceClassIconProps = {
   iconSize?: string,
 };
 
+export const getImageStreamIcon = (tag) => {
+  return _.get(tag, 'annotations.iconClass');
+};
+
+export const getImageStreamImage = (iconClass) => {
```

Comment:
There's nothing image stream specific about this. We should call this something like `getImageForIconClass` and use it in `getServiceClassImage` above.